Documentation/ceph-object-multisite.md

```diff
@@ -124,3 +241,36 @@ radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-g
 radosgw-admin zone rm --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
 radosgw-admin period update --commit --rgw-realm=realm-a --rgw-zone-group=zone-group-a --rgw-zone=zone-a
 ```
+
+When a zone is deleted, the pools for that zone are not deleted.
+
+### Deleting Pools for a Zone
+
+The Rook toolbox can delete pools. Deleting pools should be done with caution.
+
+If you do intend to deletepools you should read the following [documentation](https://docs.ceph.com/docs/master/rados/operations/pools/) on pools first.
+
+When a zone is created the following pools are created for each zone:
+```console
+$ZONE_NAME.rgw.control
+$ZONE_NAME.rgw.meta
+$ZONE_NAME.rgw.log
+$ZONE_NAME.rgw.buckets.index
+$ZONE_NAME.rgw.buckets.non-ec
+$ZONE_NAME.rgw.buckets.data
+```
+Here is an example command to delete the .rgw.buckets.data pool for zone-a.
```

Comment:
How about mentioning that the pool name has to be listed twice for verification. Otherwise, the example command looks duplicated.